packages/components/src/flyout/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as Flyout } from './flyout';
+export * from './context';
```

Comment:
Do we need to export the `context`?

Reply:
I kept that export since it was also in the "original" g2 popover component, but it's actually not used by any external package. I removed it in https://github.com/WordPress/gutenberg/pull/32197/commits/acdc90605ff14f88358bd8e8fac3626a04306fa7